File: Android/Java/UD3/Practica/app/src/main/java/com/pmdm/practica/Actividad3.java
package com.pmdm.practica;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class Actividad3 extends AppCompatActivity {

    private Button boton1;

    private Button boton2;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.actividad3);
        boton1 = findViewById(R.id.Button01);
        boton2 = findViewById(R.id.Button02);

        boton1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                irAActividad1(null);
            }
        });

        boton2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                irAActividad2(null);
            }
        });
    }

    private void irAActividad1(View view){
        Intent i = new Intent(this, MainActivity.class);
        startActivity(i);
    }

    private void irAActividad2(View view){
        Intent i = new Intent(this, Actividad2.class);
        startActivity(i);
    }
}

File: Android/Kotlin/UD3/Practica/app/src/main/java/com/pmdm/practica/Actividad3.kt
package com.pmdm.practica

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class Actividad3 : AppCompatActivity(){

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.actividad3)
        val boton1: Button = findViewById(R.id.Button01)
        val boton2: Button = findViewById(R.id.Button02)

        boton1.setOnClickListener{

        }

        boton2.setOnClickListener{

        }
    }

    private fun irAActividad1(view: View?){
        val i: Intent = Intent(this, MainActivity::class.java)
        startActivity(i)
    }

    private fun irAActividad2(view: View?){
        val i: Intent = Intent(this, Actividad2::class.java)
        startActivity(i)
    }
}